File: src/org/jetbrains/haskell/sdk/GHCDir.java
package org.jetbrains.haskell.sdk;


import org.jetbrains.haskell.util.GHCUtil;
import org.jetbrains.haskell.util.GHCVersion;

final class GHCDir {

    final String name;
    final GHCVersion version;

    GHCDir(String name) {
        this.name = name;
        this.version = GHCUtil.getVersion(name);
    }
}

File: plugin/src/org/jetbrains/haskell/sdk/GHCDir.kt
package org.jetbrains.haskell.sdk


import org.jetbrains.haskell.util.GHCUtil
import org.jetbrains.haskell.util.GHCVersion

class GHCDir(val name: String) {
    val version: GHCVersion = GHCUtil.getVersion(name)
}
